csi/controller.go

```diff
@@ -610,6 +610,14 @@ func (s *OsdCsiServer) CreateVolume(
 		if err != nil {
 			return nil, err
 		}
+
+		_, err = volumes.Update(ctx, &api.SdkVolumeUpdateRequest{
+			VolumeId: cloneResp.GetVolumeId(),
+			Labels:   getClonedPVCMetadata(locator),
```

Comment:
do we need to worry about merging with the existing labels instead of overwriting?

Reply:
volumes.Updates code suggests we don't, but I'll merge them here to be safe.

Reply:
actually, going to move this to the SDK layer to handle this more broadly. will re-request review then.